src/index.js

```diff
@@ -147,6 +147,25 @@ async function fetchTokens(password) {
   })
 }
 
+async function handle2FA() {
+  await this.deactivateAutoSuccessfulLogin()
```

Comment:
On peut utiliser `this.deactivateAutoSuccessfulLogin` et `this this.notifySuccessfulLogin` pour tous les login, même sans 2FA. Donc je les mettrais plus au niveau de la fonction `login`. Comme ça, l'utilisateur n'a pas besoin d'attendre 8s pour savoir si ses identifiants sont bons.